pyproject.toml

```diff
@@ -58,12 +58,14 @@ test = [
     "netCDF4",
     "packaging>=21.0",
     "pytest>=7.0",
-    "pytest-mpl"
+    "pytest-mpl",
+    "vcrpy>=4.3.1"
```

Comment:
This is currently not in ci/test_requirements, and so not testing on Conda in CI.

Reply:
Good catch. Fixed.